panther_analysis_tool/main.py

```diff
@@ -230,6 +231,7 @@ def load_analysis_specs(
                         fnmatch(relative_path, path_pattern)
                         for path_pattern in (
                             DATA_MODEL_PATH_PATTERN,
```

Comment:
Do we need a LUTS patch where customers can put THEIR LUTs so they would get packaged? Perhaps we change `GREYNOISE_LUTS_PATH_PATTERN` to `LUTS_PATH_PATTERN,` and nest greynoise under a `lookup_tables` folder?

Reply:
good call